typings/json-mask.d.ts

```diff
@@ -0,0 +1,14 @@
+declare module 'json-mask' {
+
+	type Dictionary<T> = { [key: string]: T };
```

Comment:
When I was talking about a dictionary type I was talking about having it shared, ie
```typescript
declare global {
	interface Dictionary<T> {
		[key: string]: T;
	}
}